src/mixins/load-keycloak-roles-mixin.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Vue } from 'vue-property-decorator'
```

Comment:
Do you think this function can move to common components in the future?

Reply:
Yes. I just followed the existing code style (eg, NR Affiliation Mixin). Personally I don't mind this as it doesn't pollute the namespace of every component with a method they won't use.

Can this be done in the future maybe?

Reply:
`vue-property-decorator` isn't a package required for Vue 3 (or Vue 2 that integrates with composition api),
but to fully move away from it would require a rewrite of A LOT of class based components 😅 

Just some thoughts!

Reply:
I think Patrick was referring to this new mixin that I created that contains a single method.

Reply:
Ah of course 😆  Sorry!